bin/pkg/list_locked_dependencies.ml

```diff
@@ -0,0 +1,113 @@
+open! Import
+open Pkg_common
+module Package_universe = Dune_pkg.Package_universe
+module Lock_dir = Dune_pkg.Lock_dir
+module Opam_repo = Dune_pkg.Opam_repo
+module Package_version = Dune_pkg.Package_version
+module Opam_solver = Dune_pkg.Opam_solver
+
+let info =
+  let doc = "List the dependencies locked by a lockdir" in
+  let man = [ `S "DESCRIPTION"; `P "List the dependencies locked by a lockdir" ] in
+  Cmd.info "list-locked-dependencies" ~doc ~man
+;;
+
+let package_deps_in_lock_dir_pp package_universe package_name ~transitive =
+  let traverse, traverse_word =
+    if transitive then `Transitive, "Transitive" else `Immediate, "Immediate"
+  in
+  let opam_package =
+    Package_universe.opam_package_of_package package_universe package_name
+  in
+  let list_dependencies which =
+    Package_universe.opam_package_dependencies_of_package
+      package_universe
+      package_name
+      ~which
+      ~traverse
+  in
+  Pp.concat
+    ~sep:Pp.newline
+    [ Pp.verbatim
```

Comment:
How come it's not `Pp.text`?

Reply:
`Pp.text` wraps the text in a way that I find hard to read. With `verbatim` the line doesn't get broken whereas switching to `Pp.text` it wraps like:
```
- Immediate dependencies of local package
   local_2.dev
   - baz.0.0.1
   - qux.0.0.1 (test only)
```
I find that hard to read because it looks to me like the subheading is `Immediate dependencies of local package` but actually the subheading is `Immediate dependencies of local package local_2.dev`.

Reply:
I think that's because you just forgot the box around the text. E.g. it should be:

```
Pp.hovbox (Pp.text ..)
```

We're getting `Pp.paragraph` to make this a little easier to remember.

Reply:
Ah cool I just found `Pp.hbox` which prevents line breaks.